app/models/drop.rb

```diff
@@ -1,10 +1,21 @@
-require 'date'
-require 'active_support/all'
-
 class Drop < ActiveRecord::Base
+
   has_many :votes, dependent: :destroy
   belongs_to :place
 
   validates :sc_track, presence: true, numericality: { only_integer: true, message: "Sorry, this isn't a valid SoundCloud link. Try again!"}
   validates :place_id, presence: { message: "Please pick a place for your drop."}
+
+  def image_from(soundcloud_track)
+    if self.soundcloud_track.artwork_url.nil?
+      self.soundcloud_track.user.avatar_url.sub! 'large', 'crop'
+    else
+      self.soundcloud_track.artwork_url.sub! 'large', 'crop'
+    end
```

Comment:
Looks like the `soundcloud_track` argument is not being used, as we are using the soundcloud_track method? Also, for instance methods you can leave out the `self.` method. Or do you prefer to use `self.` for instance methods?